java/maven/build.xml

```diff
@@ -30,8 +30,8 @@
         <javac srcdir="mavensrc" destdir="build/mavenclasses" 
             debug="${build.compiler.debug}" 
             deprecation="${build.compiler.deprecation}" 
-            source="1.5" 
-            target="1.5" 
+            source="${javac.source}" 
+            target="${javac.target}" 
```

Comment:
Introduced [NETBEANS-2639](https://issues.apache.org/jira/browse/NETBEANS-2639). More broadly, I suspect a lot of source/target level patches here are incorrect.

Reply:
There can be some issue like what happened for NETBEANS-2639. Full testing of NB is required to identify the issues (if any)